adapters/handlers/rest/authz/handlers_authz.go

```diff
@@ -76,6 +76,28 @@ func SetupHandlers(api *operations.WeaviateAPI, controller authorization.Control
 	api.AuthzGetRolesForOwnUserHandler = authz.GetRolesForOwnUserHandlerFunc(h.getRolesForOwnUser)
 }
 
+func (h *authZHandlers) AuthorizeRoleScopes(principal *models.Principal, originalVerb string, policies []authorization.Policy, roleName string) error {
```

Comment:
should we use lower case? Or is there a reason that this is public?

Reply:
good point ,it shall be private